index.js

```diff
@@ -51,6 +51,12 @@ Plugin.prototype.apply = function(compiler) {
       }
 
       var chunks = {};
+      var outputDir = this.options.path || '.';
+      var outputFilename = path.join(outputDir, this.options.filename || DEFAULT_OUTPUT_FILENAME);
+      var obj = JSON.parse(fs.readFileSync(outputFilename, 'utf8'));
+      if (obj.status == 'done') {
+          chunks = obj.chunks;
+      }
```

Comment:
ok. See if you can share a working example as a github repo so I can just clone and reproduce